app/Command/Docs.hs

```diff
@@ -132,6 +146,19 @@ pscDocsOptions = PSCDocsOptions <$> format <*> output <*> compileOutputDir <*> m
   inputFile = Opts.strArgument $
        Opts.metavar "FILE"
     <> Opts.help "The input .purs file(s)"
+  
+  globInputFile :: Opts.Parser (Maybe FilePath)
+  globInputFile = Opts.optional $ Opts.strOption $
+      Opts.long "source-globs"
+    <> Opts.metavar "FILE"
+    <> Opts.help "A file containing a line-separated list of .purs globs."
+
+  excludeFile :: Opts.Parser FilePath
+  excludeFile = Opts.strOption $
+      Opts.short 'x'
+    <> Opts.long "exclude-files"
+    <> Opts.help "Glob of .purs files to exclude from the supplied files."
+
```

Comment:
These are theoretically separate flags for separate commands, but they really are the same flag and we're duplicating this code 5 times - feels like they should be shared?